test/e2e/e2e.go

```diff
@@ -275,8 +396,12 @@ func createEFSPVCPV(c clientset.Interface, namespace, name, path string) (*v1.Pe
 }
 
 func makeEFSPVCPV(namespace, name, path string) (*v1.PersistentVolumeClaim, *v1.PersistentVolume) {
+	pv := makeEFSPvWithoutTLS(name, path)
+	return makePVCWithPV(pv, namespace, name)
+}
+
+func makePVCWithPV(pv *v1.PersistentVolume, namespace, name string) (*v1.PersistentVolumeClaim, *v1.PersistentVolume) {
```

Comment:
This function name is confusing. "With" could mean "make a PVC *along* with a PV".

I suggest separating the binding code below (setting volumeName of a given PVC and claimRef of a given PV) to a separate function.